src/js/components/ConfigurationMapSection.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const ConfigurationMapSection = (props) => {
+  return (
+    <div className="configuration-map-section pod flush-top flush-right flush-left">
```

Comment:
@jfurrow  just curious why we don't include `pod flush-top flush-right flush-left` in `configuration-map-section` class?